test/e2e/yaml/windows/non-hpc-pod.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: non-hpc-pod
+  labels:
+    app: non-hpc
```

Comment:
suggest: Can we provide any more descriptive name than 'non-hpc-pod' ? What about something just like ebpf-test-pod, or anything else?